enterprise_access/apps/api/tasks.py

```diff
@@ -1,26 +1,94 @@
 """
 Celery tasks for Enterprise Access API.
 """
+import logging
 
 from celery import shared_task
 from celery_utils.logged_task import LoggedTask
+from django.conf import settings
 
-from enterprise_access.apps.subsidy_request.constants import SubsidyTypeChoices
+from enterprise_access.apps.api_client.braze_client import BrazeApiClient
+from enterprise_access.apps.api_client.lms_client import LmsApiClient
+from enterprise_access.apps.subsidy_request.constants import (
+    ENTERPRISE_BRAZE_ALIAS_LABEL,
+    SUBSIDY_TYPE_CHANGE_DECLINATION,
+    SubsidyRequestStates,
+    SubsidyTypeChoices
+)
 from enterprise_access.apps.subsidy_request.models import CouponCodeRequest, LicenseRequest
 
+logger = logging.getLogger(__name__)
+
+
+def _aliased_recipient_object_from_email(user_email):
+    """
+    Returns a dictionary with a braze recipient object, including
+    a braze alias object.
```

Comment:
Docs need a return & args statement.

Reply:
i need to find copy pasta with better doc strings ;)

Reply:
added args. return was already in docstring